lib/main.dart

```diff
@@ -1,113 +1,151 @@
-          // how it positions its children. Here we use mainAxisAlignment to
-          // center the children vertically; the main axis here is the vertical
-          // axis because Columns are vertical (the cross axis would be
-          // horizontal).
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            Text(
-              'You have pushed the button this many times:',
-            ),
-            Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.headline4,
+      debugShowCheckedModeBanner: false,
+      title: 'Business Card',
+      home: Scaffold(
+        body: Container(
+          child: SafeArea(
+            top: false,
+            child: Container(
+              decoration: BoxDecoration(
+                image: DecorationImage(
+                    image: AssetImage(
+                      "assets/images/cardbackground.png",
+                    ),
+                    fit: BoxFit.fill),
+              ),
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                  Container(
+                    decoration: BoxDecoration(
+                      shape: BoxShape.circle,
+                      boxShadow: [
+                        BoxShadow(
+                          blurRadius: Constants.avatarBlurShadow,
+                          color: Constants.avatarShadowColor,
+                          spreadRadius: 1,
+                        ),
+                      ],
+                    ),
+                    child: CircleAvatar(
+                      backgroundImage: AssetImage(
+                        'assets/images/me.jpg',
+                      ),
+                      radius: Constants.avatarRadius,
+                    ),
+                  ),
+                  SizedBox(
+                    height: Constants.avatarAndPersonNameSizeBoxHeight,
+                  ),
+                  Text(
+                    'Kevin Guillen',
+                    style: TextStyle(
+                      fontSize: Constants.personNameFontSize,
+                      fontWeight: FontWeight.bold,
+                      color: Constants.personNameTextColor,
+                      letterSpacing: Constants.letterSpacing,
+                    ),
+                  ),
+                  SizedBox(
+                    height: Constants.personNameAndPositionSizeBoxHeight,
+                  ),
+                  Text(
+                    'Flutter Developer Trainee',
+                    style: TextStyle(
+                      fontSize: Constants.positionFontSize,
+                      fontWeight: FontWeight.bold,
+                      color: Constants.positionTextColor,
+                      letterSpacing: Constants.letterSpacing,
+                    ),
+                  ),
+                  Column(
```

Comment:
Is this column needed ? you could omit the column and just use the divider instead right ?

Reply:
done